tests/ExceptionHandlerTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Fuzz\ImageResizer\Tests;
+
+use Fuzz\ImageResizer\ExceptionHandler;
+use InvalidArgumentException;
+use LogicException;
+use Mockery;
+use RuntimeException;
+
+class ExceptionHandlerTest extends ImageResizerTestCase
+{
+	/**
+	 * @runInSeparateProcess
+	 */
+	public function testItReturnsNiceErorOnInvalidArgumentException()
```

Comment:
I prefer `testInvalidArgumentException`. In general, I also try to document each test case with an extremely brief sentence that someone like a member of the product team could make sense of.